spyglass/server.py

```diff
@@ -26,34 +26,42 @@ class StreamingServer(socketserver.ThreadingMixIn, server.HTTPServer):
 
 def run_server(bind_address, port, output):
     class StreamingHandler(server.BaseHTTPRequestHandler):
+        streamURL = '/stream'
+        snapshotURL = '/snapshot'
+        
         def do_GET(self):
-            if self.path == '/stream.mjpg':
-                self.send_response(200)
-                self.send_header('Age', 0)
-                self.send_header('Cache-Control', 'no-cache, private')
-                self.send_header('Pragma', 'no-cache')
-                self.send_header('Content-Type', 'multipart/x-mixed-replace; boundary=FRAME')
-                self.end_headers()
-                try:
-                    while True:
-                        with output.condition:
-                            output.condition.wait()
-                            frame = output.frame
-                        self.wfile.write(b'--FRAME\r\n')
-                        self.send_header('Content-Type', 'image/jpeg')
-                        self.send_header('Content-Length', len(frame))
-                        self.end_headers()
-                        self.wfile.write(frame)
-                        self.wfile.write(b'\r\n')
-                except Exception as e:
-                    logging.warning(
-                        'Removed streaming client %s: %s',
-                        self.client_address, str(e))
+            if self.path in [self.streamURL, self.snapshotURL]:
+                self.startStream(self.path)
             else:
                 self.send_error(404)
                 self.end_headers()
 
-    logger.info("Server listening on http://%s:%d/stream.mjpg", bind_address, port)
+        def startStream(self, url):            
```

Comment:
AFAIK the Python convention would be `start_stream`

Reply:
The method doesn't make sense anyway atm.
I rewrote the method and forgot to delete it again as the code can go directly into do_GET